platform/admin/core/admin-core-insecuredefaults/pom.xml

```diff
@@ -78,10 +108,16 @@
                 <configuration>
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
-                        <Embed-Dependency>common-system, alerts</Embed-Dependency>
-                        <Import-Package>*</Import-Package>
-                        <Export-Package>
-                        </Export-Package>
+                        <Embed-Dependency>
+                            common-system,
+                            alerts,
+                            quartz,
+                            c3p0,
+                            camel-core,
```

Comment:
❓ Do all of these need to be embedded? I would think that at the very least the classes in use from the `camel-core` module were exposed through services.

Reply:
I'm trying to import camel-core and maybe camel-quartz too but I wasn't able to. I will continue trying.

Reply:
I don't actually need `camel-core` but I want' able to not embed the rest. `camel-quartz` -> `quartz` -> `c3p0`

Reply:
I take that back I do need `camel-core`.
I get the following error without it:
```Caused by: org.apache.camel.ResolveEndpointFailedException: Failed to resolve endpoint: quartz2://scheduler/deletionTimer?cron=0+16+15+26+10+%3F+2017 due to: No component found with scheme: quartz2```